ui/components/app/Analysis/ExpiredJWTCard.tsx

```diff
@@ -39,9 +40,43 @@ const columns: GridColDef[] = [
     },
     width: 200,
   },
-  { field: 'Src IP', headerName: 'Src IP' },
   { field: 'URI', headerName: 'URI', width: 400 },
-  { field: 'Dest IP', headerName: 'Dest IP' },
+  {
+    field: 'Src IP',
+    headerName: 'Source IP',
+    width: 175,
+    renderCell(params: GridRenderCellParams<string>) {
+      return (
+        <>
+          <IPAddress
+            className='mr-2'
+            size={30}
+            isoCode={params.row.srcCountryCode}
+            lat={params.row.srcLat}
+            long={params.row.srcLong}
+            address={params.value ?? '-'}
+          />
+        </>
```

Comment:
You need `<>` which is a syntax shortcut for `React.Fragment` when you have multiple elements at the top-level of a JSX expression, such as `return <button/><button/>` which doesn't work without `<>`. However, this is just an IPAddress, so you don't need the Fragment wrapper.

```suggestion
        <IPAddress
          className='mr-2'
          size={30}
          isoCode={params.row.srcCountryCode}
          lat={params.row.srcLat}
          long={params.row.srcLong}
          address={params.value ?? '-'}
        />
```